src/olympia/constants/activity.py

```diff
@@ -1015,6 +1015,31 @@ class NEEDS_HUMAN_REVIEW_CINDER(NEEDS_HUMAN_REVIEW_AUTOMATIC):
     review_queue = True
 
 
+class AUTO_REJECT_VERSION_AFTER_DELAY_EXPIRED(_LOG):
+    # takes add-on, version, reviewtype
+    id = 189
+    action_class = 'reject'
+    format = _('{addon} {version} rejected automatically after delay expired.')
+    short = _('Rejected automatically after delay expired')
```

Comment:
These `short` descriptions are a little non-short 😬 
I don't have a great alternative... is "Rejected automatically after expiration" too vague? (and only a little shorter anyway).
Or to reword completely to match how we label the delayed rejection event, "Scheduled automatic rejection"?

Reply:
The other strings to be similarly adjusted if you agree.

Reply:
Semi-nit comments I guess, but I can't remember everywhere we expose `short` as a string in the UI.

Reply:
I like "Scheduled automatic rejection", but when I added it, I started to wonder if this can be read as "[someone] Scheduled [an] automatic rejection".  If we follow the other labels ("Rejection scheduled"), how about we use "Scheduled rejection due" or similar? What's a good way to express that?

Reply:
After some discussion on slack, we will be moving back to the original copy.